File: perl/tests/002-session-status.pl
my $normal    = &NormalHandle;
my $normal_id = $normal->session_status();

print "NORMAL_ID= ${normal_id}\n" if $DEBUG;

die "Value returned is not an ID\n" unless defined($normal_id);
die "ID returned by session_status (${normal_id}) isn't the same as the ID of API (",$normal->current_user_id(),")." if $normal_id != $normal->current_user_id();

1;

File: ruby/tests/002-session-status.rb
normal    = normal_handle
normal_id = normal.session_status()

puts "NORMAL_ID= #{normal_id}" if @debug

raise "Value returned is not an ID" unless normal_id.present? && normal_id.is_a?(Integer)
raise "ID returned by session_status (#{normal_id}) isn't the same as the ID of API (#{normal.current_user_id})." if normal_id != normal.current_user_id()

true
